spoon-client/src/main/java/com/squareup/spoon/Spoon.java

```diff
@@ -1,23 +1,19 @@
 package com.squareup.spoon;
 
 import android.app.Activity;
-import android.content.Context;
 import android.graphics.Bitmap;
 import android.graphics.Canvas;
 import android.os.Looper;
-import android.util.DisplayMetrics;
 import android.util.Log;
-import java.io.BufferedOutputStream;
-import java.io.File;
-import java.io.FileOutputStream;
-import java.io.IOException;
-import java.io.OutputStream;
+import android.view.View;
+
+import java.io.*;
```

Comment:
Please don't use star imports. I'm pretty sure checkstyle fails the build because of this as well.